---
layout: page
title: Software
permalink: /software/
---

# Bioinformatics tools
* [bpipe](http://docs.bpipe.org/) - A tool for running and managing bioinformatics pipelines

# Transcriptomics
* [Corset](https://github.com/Oshlack/Corset/wiki) - Go from a de novo transcriptome assembly to gene-level counts.
* [Lace](https://github.com/Oshlack/Lace/wiki) - A tool for constructing superTranscripts: a single linear representation of a gene built from constituent transcripts, retaining all exonic sequence in transcriptional order.
* [Necklace](https://github.com/Oshlack/necklace/wiki) - Combine reference and assembled transcriptomes for RNA-Seq analysis.
* [GOseq](http://bioconductor.org/packages/release/bioc/html/goseq.html) - Bioconductor package: Gene Ontology analyser for RNA-seq and other length biased data
* [Splatter](https://github.com/Oshlack/splatter) - Bioconductor package for the simple simulation of single-cell RNA-seq count data.
* [clinker](https://github.com/Oshlack/Clinker) - A gene fusion visualiser tool.

# Cancer
* [JAFFA](https://github.com/Oshlack/JAFFA/wiki) - A multi-step pipeline that takes either raw RNA-Seq reads, or pre-assembled transcripts, then searches for gene fusions.
* [AllSorts](https://github.com/Oshlack/AllSorts) - An Acute Lymphoblasitc Leukemia B-Cell classifier

# Genomics
* [Cpipe](https://melbournegenomics.github.io/) - An exome sequencing pipeline developed for analysis of exome and targeted sequencing data that is specialised for clinical purposes.
* [Ximmer](http://ximmer.org/) - A tool for evaluating, running and viewing results of CNV callers for exome and targeted sequencing data
* [Schism](https://github.com/ssadedin/schism) - A tool for discovering novel, complex, pathogenic structural variation in whole 
  genome sequencing data.

# Methylation
* [missMethyl](http://www.bioconductor.org/packages/release/bioc/html/missMethyl.html) - Bioconductor package for analysing data from Illumina's methylation arrays
